docs/sources/tempo/traceql/_index.md

```diff
@@ -93,11 +94,19 @@ The following table shows the current available scoped intrinsic fields:
 | `link:spanID`           | string      | link span id using hex string                                   | `{ link:spanID = "0000000000000001" }` |
 | `link:traceID`          | string      | link trace id using hex string                                  | `{ link:traceID = "1234567890abcde" }` |
 
-{{< admonition type="note" >}}
-`trace:duration`, `trace:rootName`, and `trace:rootService` are trace-level intrinsics and will be the same for all spans in the same trace. Additionally,
-these intrinsics are significantly more performant because they have to inspect much less data then a span-level intrinsic. They should be preferred whenever
-possible to span-level intrinsics.
-{{% /admonition %}}
+`trace:duration`, `trace:rootName`, and `trace:rootService` are trace-level intrinsics and are the same for all spans in the same trace.
+Additionally, these intrinsics are significantly more performant because they have to inspect much less data then a span-level intrinsic.
+They should be preferred whenever possible to span-level intrinsics.
+
+You may have a time when you want to search by a trace-level intrinsic instead.
+For example, using `span:name` looks for the names of spans within traces.
+If you want to search by a trace name of `perf`, use `trace:rootName` to match against trace name.
+
+This example searches all Kubernetes clusters called `service-name` that have a span with a root name of including `perf`.
+
+```
+{ .k8s.cluster.name="service-name" && rootName !~ ".*perf.*"}
```

Comment:
let's use the full name of intrinsics where possible and remove unscoped attributes.

Maybe:
```
{ resource.k8s.cluster.name="service-name" && trace:rootName !~ ".*perf.*"}
```

Reply:
Thank you for the suggestions. I'll make the update. 
```suggestion
{ resource.k8s.cluster.name="service-name" && trace:rootName !~ ".*perf.*"}
```